src/App.js

```diff
@@ -73,6 +93,12 @@ export default function App() {
         </li>
       </ol>
       If done right, each of these spells will summon a magical gift that will help you on your journey...
+      <MagicImage
+        currentArrayPosition={currentArrayPosition}
+        currentBackgroundImage={currentBackgroundImage}
+        isRoomEnlighten={isRoomEnlighten}
+        changeEnlighten={changeEnlighten}
```

Comment:
You are passing 4 props here although your `MagicImage` component only receives two: `isRoomEnlighten` and `currentBackgroundImage`. I would recommend to remove the props you don't need :)
```suggestion
        currentBackgroundImage={currentBackgroundImage}
        isRoomEnlighten={isRoomEnlighten}
```